Documentation/cmdref/cilium-agent.md

```diff
@@ -53,13 +53,15 @@ cilium-agent [flags]
       --disable-iptables-feeder-rules strings         Chains to ignore when installing feeder rules.
       --egress-masquerade-interfaces string           Limit egress masquerading to interface selector
       --enable-auto-protect-node-port-range           Append NodePort range to net.ipv4.ip_local_reserved_ports if it overlaps with ephemeral port range (net.ipv4.ip_local_port_range) (default true)
+      --enable-bpf-masquerade                         Masquerade packets from endpoints leaving the host with BPF instead of iptables
       --enable-endpoint-health-checking               Enable connectivity health checking between virtual endpoints (default true)
       --enable-endpoint-routes                        Use per endpoint routes instead of routing via cilium_host
       --enable-external-ips                           Enable k8s service externalIPs feature (requires enabling enable-node-port) (default true)
       --enable-health-checking                        Enable connectivity health checking (default true)
       --enable-host-port                              Enable k8s hostPort mapping feature (requires enabling enable-node-port) (default true)
       --enable-host-reachable-services                Enable reachability of services for host applications (beta)
       --enable-hubble                                 Enable hubble server
+      --enable-ip-masq-agent                          Enable BPF ip-masq-agent
```

Comment:
We have:
 `--masquerade` which enables masquerade using iptables
 `--enable-bpf-masquerade` which enable masquerades using bpf.
 and now we have
 `--enable-ip-masq-agent` which enables ip-masq-agent in... bpf or iptables? (The flag name is not consistent with other flags)

Reply:
I could rename it to `--enable-bpf-ip-masq-agent`, but IMHO the `bpf` is redundant here, as we are getting rid of iptables. I think in the future we will be able to replace `--enable-bpf-masquerade` with `--masquerade`.